client/client-multi/client-cpp/impl/logging/LoggingUtils.cpp

```diff
@@ -0,0 +1,809 @@
+/*
+ * Copyright 2014-2016 CyberVision, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "kaa/logging/LoggingUtils.hpp"
+
+#include <iomanip>
+#include <sstream>
+#include <iterator>
+
+#include <boost/format.hpp>
+
+namespace kaa {
+
+#define KVSTRING(K,V) "\"" #K "\": \"" << V << "\""
```

Comment:
This macro isn't used anywhere, I suggest you to undef it at the end of the file